lib/WP_Auth0_WooCommerceOverrides.php

```diff
@@ -1,29 +1,40 @@
 <?php
 
 class WP_Auth0_WooCommerceOverrides {
-
 	protected $plugin;
 
-	public function __construct( WP_Auth0 $plugin ) {
+	public function __construct( WP_Auth0 $plugin, $options ) {
 		$this->plugin = $plugin;
+		$this->options = $options;
 	}
 
 	public function init() {
 		add_filter( 'woocommerce_checkout_login_message', array( $this, 'override_woocommerce_checkout_login_form' ) );
 		add_filter( 'woocommerce_before_customer_login_form', array( $this, 'override_woocommerce_login_form' ) );
 	}
 
+	private function render_login_form( $redirect ) {
+		$this->plugin->render_auth0_login_css();
+		if ($this->options->get('auto_login',false)) {
+			$redirection_after = site_url( $redirect );
```

Comment:
Feels a little confusing to call the param `$redirect` and then assume it's a path. Not to get too in the weeds for a private function but I'd suggest renaming this to `$site_path` or something more clear.

Reply:
This might be a moot point with the checkout/account URL getting below.

Reply:
done